templates/first_stage_script.sh

```diff
@@ -10,7 +10,7 @@ which gpg >/dev/null || exit 1
 while [[ -z $(curl -k -s {{.MayuURL}}) ]] ; do sleep 1;  done
 
 INFOPUSHER_IMAGE="quay.io/giantswarm/mayu-infopusher:latest"
-TMP_HELPER="docker run --net=host --privileged=true -v /sys:/sys -v /dev:/dev -it $INFOPUSHER_IMAGE"
+TMP_HELPER="docker run --net=host --privileged=true -v /sys:/sys -v /dev:/dev  $INFOPUSHER_IMAGE"
```

Comment:
`-it` for service does not work because `systemd` unit does not have `tty` and this will end by error